test/framework/src/main/java/org/elasticsearch/index/mapper/MapperServiceTestCase.java

```diff
@@ -91,6 +91,12 @@ protected final DocumentMapper createDocumentMapper(XContentBuilder mappings) th
         return createMapperService(mappings).documentMapper();
     }
 
+    protected final DocumentMapper createDocumentMapper(String mappings) throws IOException {
```

Comment:
This one is used by some tests in DocumentParserTests which load mappings defined in an external file.  I'm also open to just rewriting it so that it's all done in java code.

Reply:
Its cool.